lumicks/pylake/scan.py

```diff
@@ -48,7 +48,7 @@ def num_frames(self):
 
     @property
     def lines_per_frame(self):
-        return self._json["scan volume"]["scan axes"][1]["num of pixels"]
+        return self._num_pixels[self._scan_order[1]]
```

Comment:
(see comment about splitting commit)

Reply:
I've split them now. One specifically for the introduction of `_num_pixels`, then one that changes a few calls to use that attribute, and then the last one which gets rid of the `fast_axis` attribute.

Reply:
First commit in this PR looks wrong now :)

Reply:
You mean introduce two private functions and not use them? Same. I'd preferably squash 1 and 2; but the "see comment about splitting commit" here confused me and I figured you wanted a break between introducing the function and replacing it here.

Reply:
Agree with squashing the first two commits. The third commit `confocal: remove attribute _fast_axis_metadata` is exactly what I asked for. 👍🏽

Reply:
👍